test/functional/testlib.bash

```diff
@@ -3169,6 +3169,7 @@ get_next_available_id() { # swupd_function
 	id=$((id+1))
 	test_dir=$(basename "$(realpath "$test_dir")")
 	case "$test_dir" in
+		autoupdate) group=AUP;;
```

Comment:
Why did you change the code?

Reply:
It has to three letter. Thats why. Couldnt think of any other

Reply:
Just use what we have defined before AUT. I don't know why @castulo picked that but we already have documentation using that name

Reply:
okay

Reply:
Done